ansible/roles_ocp_workloads/ocp4_workload_dso/tasks/gogs.yml

```diff
@@ -76,7 +85,11 @@
   uri:
     url: http://{{ ocp4_dso_gogs_urlprefix }}.{{ route_subdomain }}/api/v1/admin/users
     method: POST
-    body: "{'login_name': '{{ ocp4_dso_gogs_user }}', 'username': '{{ ocp4_dso_gogs_user }}', 'email': '{{ ocp4_dso_gogs_user }}@gogs.com', 'password': '{{ ocp4_dso_gogs_user_password }}'}"
+    body: >-
+      {'login_name': '{{ ocp4_dso_gogs_user }}',
+      'username': '{{ ocp4_dso_gogs_user }}',
+      'email': '{{ ocp4_dso_gogs_user }}@gogs.com',
+      'password': '{{ ocp4_dso_gogs_user_password }}'}
```

Comment:
Better way to handle this, the ansible `uri` module natively handles a dict for the `body`:

```
- name: create gogs user
  uri:
    url: http://{{ ocp4_dso_gogs_urlprefix }}.{{ route_subdomain }}/api/v1/admin/users
    method: POST
    body:
      login_name: '{{ ocp4_dso_gogs_user }}'
      username: '{{ ocp4_dso_gogs_user }}'
      email: '{{ ocp4_dso_gogs_user }}@gogs.com'
      password: '{{ ocp4_dso_gogs_user_password }}'
    body_format: json
    user: "{{ ocp4_dso_gogs_admin_user }}"
    password: "{{ ocp4_dso_gogs_admin_password }}"
    status_code: 200,201
    force_basic_auth: true
  when: user_result.status != 200
```